schemes/fingerprint-server-api.yaml

```diff
@@ -131,7 +189,10 @@ components:
         visits:
           type: array
           items:
-            $ref: '#/components/schemas/Visit'
+            allOf:
```

Comment:
I had problems with code generation using `allOf`, at least for python SDK.
Better to check it.

Reply:
I see, I will check it out. So far I've tested if for Go SDK and after small changes in the template it worked.